src/components/Game/DeadScreen.tsx

```diff
@@ -30,9 +30,9 @@ function DeadScreen(): JSX.Element {
                         Du rakk dessverre ikke fram til mål i tide
                     </Heading2>
                     <Image
-                        className="mb-16 mt-10"
+                        className="xl:mb-16 xl:mt-10 lg:mt-5 lg:mb-5 md:mt-3 md:mb-3 xl:w-[500px] lg:w-80 md:w-48"
```

Comment:
Samme her
className="mt-3 mb-3 lg:mt-5 lg:mb-5 xl:mb-16 xl:mt-10 w-48 lg:w-80 xl:w-[500px]"

Ikke en stor greie, siden vi aldri uansett er under md, men greit å ha margins for alle likevel